src/shared/sam/debugger/awsSamDebugger.ts

```diff
@@ -307,173 +313,3 @@ export function createDirectInvokeSamDebugConfiguration(
 
     return response
 }
-/**
- * Validates debug configuration properties.
- */
-function validateConfig(folder: vscode.WorkspaceFolder | undefined, config: AwsSamDebuggerConfiguration): boolean {
```

Comment:
This logic was enhanced in https://github.com/aws/aws-toolkit-vscode/pull/1014 . Now move it to `AwsSamDebugConfigurationValidator` .